inc/theme-hooks.php

```diff
@@ -125,3 +125,123 @@ function alcatraz_output_footer_bottom() {
 		);
 	}
 }
+
+
+add_filter( 'alcatraz_primary_sidebar', 'alcatraz_sidebar_nav_menu', 5);
+/**
+ * Output the sidebar nav.
+ *
+ * @since  1.0.0
+ */
+
+function alcatraz_sidebar_nav_menu() {
+
+	global $post;
+
+	$args = array();
+
+	//defaults
+	$defaults = array(
+		'show_all'         => true,
+		'show_on_home'     => false,
+		'show_empty'       => false,
+		'sort_by'          => 'menu_order',
+		'a_heading'        => false,
+		'title'            => ''
+	);
+
+	$args = wp_parse_args( (array)$args, $defaults );
+
+	// Get clean param values.
+	$show_all         = $args['show_all'];
+	$show_on_home     = $args['show_on_home'];
+	$show_empty       = $args['show_empty'];
+	$sort_by          = $args['sort_by'];
+	$a_heading        = $args['a_heading'];
+	$title            = $args['title'];
+
+	if ( is_search() || is_404() ) {
```

Comment:
I think we should abstract a function `alcatraz_get_section_nav()` that will always build and return the section nav HTML, then our `alcatraz_output_section_nav()` function can just be a series of checks to see whether we should output anything and then call `echo alcatraz_get_section_nav()`. See Carrie's recent implementation of `alcatraz_output_social_icons()` vs. `alcatraz_get_the_social_icons()` for a good example to follow.